app/controllers/content_items_controller.rb

```diff
@@ -89,17 +91,20 @@ def set_expiry
   end
 
   def set_up_navigation
-    @navigation = NavigationType.new(@content_item.content_item)
-
     # Setting a variant on a request is a type of Rails Dark Magic that will
     # use a convention to automagically load an alternative partial, view or
     # layout.  For example, if I set a variant of :taxonomy_navigation and we render
     # a partial called _breadcrumbs.html.erb then Rails will attempt to load
     # _breadcrumbs.html+taxonomy_navigation.erb instead. If this file doesn't exist,
     # then it falls back to _breadcrumbs.html.erb.  See:
     # http://edgeguides.rubyonrails.org/4_1_release_notes.html#action-pack-variants
-    if @navigation.should_present_taxonomy_navigation?
-      request.variant = :taxonomy_navigation
+    if should_present_universal_navigation?
+      request.variant = :universal_navigation
```

Comment:
Are we sure about using variants? I would've expected that the universal navigation is a single template, not variants on the existing per-format templates.

Reply:
That might be what we end up with. For now there's still a few too many unknowns. While content will all be in two columns, there will be differences eg Guides with their pagination and page links at top vs Publications that have metadata, vs Answers which have neither.

Reply:
Cool. I think the dream would be a government-frontend that doesn't know anything about `schema_name` and `document_type` and determines layout dependent on the [features it detects](https://www.joezimjs.com/javascript/feature-detection-vs-browser-detection) (not sure if that's 100% possible though).

Reply:
Strongly agree about content itself determining features, been wanting that for a while.

Keen not to focus on global layout and template though, prefer adaptable components that can handle whatever is given to them. End result ought to be similar.